labels-en.json

```diff
@@ -17,5 +17,7 @@
     "dropdown-select": "Select",
     "kids-age-question": "3. How many kids do you have aged 5-18?",
     "kids-age-desc": "More programs may be available depending on your answer.",
-    "show-nearby-programs": "Show nearby programs"
+    "show-nearby-programs": "Show nearby programs",
```

Comment:
Since you are adding show-nearby-programs to the English labels, wouldn't you need that in the Spanish labels file as well?

Reply:
this isn't a new label, just a comma addition. We need to move all strings used in py file to this one, and that can be taken up later.